pkg/operator/hypershift.go

```diff
@@ -240,8 +240,9 @@ func parseNamespacedName(namespacedName string) string {
 
 func getMachineConfigFromConfigMap(config *corev1.ConfigMap) (*mcfgv1.MachineConfig, error) {
 	scheme := runtime.NewScheme()
-	//nolint:errcheck
-	mcfgv1.Install(scheme)
+	if err := mcfgv1.Install(scheme); err != nil {
+		return nil, err
+	}
```

Comment:
while this change per se is correct, I prefer to set up the schema once, not per each function call. Even if the function is expected to be called once (which doesn't seem the case?). This probably just means this code need further work besides this PR